spec/migrations/20220223095704_remove_col3_from_miq_widget_set_spec.rb

```diff
@@ -0,0 +1,79 @@
+require_migration
+
+describe RemoveCol3FromMiqWidgetSet do
+  let(:widget_stub) { migration_stub(:MiqWidgetSet) }
+
+  migration_context :up do
+    it "Moving col3 widgets to col2 and removing col3" do
+      data = {:col1 => [1, 2, 3], :col2 => [4, 5, 6], :col3 => [7, 8, 9]}
+      set_data = {:col1 => data[:col1], :col2 => ((data[:col2] || []) + (data[:col3] || [])).uniq.compact}
+      widget = widget_stub.create(:set_data => set_data)
```

Comment:
Maybe I'm reading this wrong, but I don't think this should be in the test (nor in the other places).  This is effectively changing the data immediately in the test and setting it to set_data, and not actually exercising the migration code.

Instead I think it should be:

```suggestion
      data = {:col1 => [1, 2, 3], :col2 => [4, 5, 6], :col3 => [7, 8, 9]}
      widget = widget_stub.create(:set_data => data)
```

or just embed it directly:

```suggestion
      widget = widget_stub.create(:set_data => {:col1 => [1, 2, 3], :col2 => [4, 5, 6], :col3 => [7, 8, 9]})
```